ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.18__multi-nightly.yaml

```diff
@@ -1439,6 +1439,7 @@ tests:
   steps:
     cluster_profile: equinix-ocp-metal-qe
     env:
+      ADDITIONAL_NTP_SERVERS: "true"
```

Comment:
Being this a job using DHCP, the nodes will also get the NTP servers from the DHCP server, right? Should we test on a static network scenario instead?

Reply:
Here we are not testing syncing with the external NTP servers with this PR.
It is only to test that ntp servers are indeed passed using install-config.yaml file. I will write a separate test in ginko to check that with a successful cluster install.